src/index.jsx

```diff
@@ -2,25 +2,39 @@ import 'react-hot-loader/root'; // This needs to be imported before React.
 import React from 'react';
 import { render } from 'react-dom';
 import * as Sentry from '@sentry/react';
+import axios from 'axios';
 import { get } from 'lodash-es';
 
 import 'normalize.css';
 import '../fonts/fonts.css';
 import './styles/globalStyles.css';
 
-import { sentryDsn } from './constants/apiKeys';
 import pjson from '../package.json';
 import App from './App';
 
-if (!__DEV__) {
-  Sentry.init({
-    dsn: sentryDsn,
-    release: `frontend@${get(pjson, 'release')}`,
-  });
-}
+axios
+  .get(`${__houston_url__}/api/v1/site-settings/main/sentryDsn`)
+  .then(response => {
+    if (__DEV__) return;
```

Comment:
Would we want to prevent the axios call if (__DEV__)?

Reply:
We have two settings in houston `SENTRY_DSN_DEVELOPMENT` and `SENTRY_DSN_PRODUCTION` so I assume we want to use sentry whether it's development or not.  Let's remove `if (__DEV__) return;`

Reply:
We were only initializing it in dev before:
https://github.com/WildMeOrg/codex-frontend/blob/a0a9a029c233a381f241a63d95031f6363e52c6b/src/index.jsx#L15-L20

Reply:
Yes I know, but looking at the houston settings `SENTRY_DSN_DEVELOPMENT` and `SENTRY_DSN_PRODUCTION`, it's clear we want to send to sentry even when development mode is true. That's why I'm changing it. Houston is also sending errors to sentry when development mode is true.